manifest.php

```diff
@@ -32,7 +32,7 @@
 	'label' => 'LTI library',
     'description' => 'TAO LTI library and helpers',
     'license' => 'GPL-2.0',
-    'version' => '1.12.0',
+    'version' => '1.13.0',
```

Comment:
Should dependency be raised to ensure ExceptionInterpreterService already exists?